contracts/smart-account/modules/SessionValidationModules/ERC20SessionValidationModule.sol

```diff
@@ -28,7 +28,13 @@ contract ERC20SessionValidationModule {
         bytes32 _userOpHash,
         bytes calldata _sessionKeyData,
         bytes calldata _sessionKeySignature
-    ) external pure returns (bool) {
+    ) external pure override returns (bool) {
+        require(
+            bytes4(_op.callData[0:4]) == EXECUTE_SELECTOR ||
```

Comment:
I think we should start using custom errors in Solidity. It would help reduce deployment costs and be more actionable for the client since they can be parameterized.
https://soliditylang.org/blog/2021/04/21/custom-errors/

Reply:
also, is `execute` expected to be called more often than `execute_xxx`? If not let's switch the order in which the selector is compared so that the more common case is evaluated first.

Reply:
1) yep, we do use custom errors and it was actually my suggestion :) however, in this specific case the revert message is appended to AA23 message by EP
```
revert FailedOp(opIndex, string.concat("AA23 reverted: ", revertReason));
```
so no reason to use custom error here.

2) fair note, will do